### ISSUE TITLE
[tcp] bus fault error when running test-tcp6-client.js on K64F

### ISSUE THREAD

**Issue by qiaojingx** [01org/zephyr.js#1491]
### Description

This is a regression issue. BUS FAULT error occurs when running test-tcp6-client.js on K64F.

### Test Code

[test-tcp6-client.js ](https://github.com/01org/zephyr.js/blob/master/tests/test-tcp6-client.js);

### Steps to Reproduction

1.  `make BOARD=frdm_k64f JS=tests/test-tcp6-client.js`
2. `cp outdir/frdm_k64f/zephyr.bin /media/<username>/MBED/`
3.  Add soft router and IPv6 address on linux:
```
ip -6 route add 2001:db8::/64 dev eno1
ip -6 addr add 2001:db8::2/64 dev eno1
```
4. `python tests/tools/test-tcp6-server.py`

### Actual Result

BUS FAULT error occurs after connection close
![selection_139](https://user-images.githubusercontent.com/19813546/29602192-b1384a26-8811-11e7-8780-da4ff25b7d70.png)
d.
 
### Expected Result

No BUS FAULT error. 

### Test Builds

Branch | Commit Id | Target Device | Test Date | Result
---|---|---|---|---
master | 2439525 | FRDM_K64F | Aug 23, 2017 | Pass
master | 40f4274 - 06cce05 | FRDM_K64F | Aug 23, 2017 | Fail

**Comment by grgustaf** [comment]
Reproduced this and unfortunately it's not fixed with my latest patches.

**Comment by grgustaf** [comment]
Fixed in #1505

**Comment by qiaojingx** [comment]
Verified with commit a0b1217 on FRDM-K64F. This issue is fixed.
